tests/phpunit/tests/test-admin.php

```diff
@@ -161,4 +179,29 @@ public function test_scripts_in_user_profile() {
 		$scripts = array( 'pll_ajax_backend', 'user', 'css' );
 		$this->_test_scripts( $scripts );
 	}
+
+	public function test_remove_customize_submenu_with_block_base_theme() {
```

Comment:
I would like to test also the case of a non block theme.
However, I can't require twice the '**_wp-admin/menu.php_**'.
And I need it to be loaded at that time. 

If you have any ideas about this.

Reply:
Is it not possible in another method? Like you do here.

Reply:
Requiring 'wp-admin/menu.php' again is blocking me because I can't declare the same functions twice.

Reply:
So basically you need it for `$submenu` to be filled.
What about this?

```php
require_once ABSPATH . 'wp-admin/menu.php';
$original_submenu = $submenu;

do_action( 'admin_menu', '' );

$this->assertFalse( in_array( 'customize', array_merge( ...array_values( $submenu['themes.php'] ) ), true ) );

// Non block theme.
// Here your code to test a non block theme.

$submenu = $original_submenu;

do_action( 'admin_menu', '' );

$this->assertFalse( in_array( 'customize', array_merge( ...array_values( $submenu['themes.php'] ) ), true ) );

$submenu = $original_submenu;
```